routes.js

```diff
@@ -4,6 +4,20 @@ module.exports = [
   {
     method: 'GET',
     path: '/',
+    config: {
+      plugins: {
+        'hapi-negotiator': {
+          mediaTypes: {
+            'text/html': function (req, reply) {
+              reply('<h1>HTML Response</h1>').header('content-type', 'text/html');
+            },
+            'application/vnd.api+json': function (req, reply) {
+              reply('"{"response": "JSONAPI"}"').header('content-type', 'application/vnd.api+json');
+            }
+          }
+        }
+      }
+    },
     handler: function (req, reply) {
       reply('OK');
     }
```

Comment:
Just a note:

If an acceptable media type is not found, hapi-negotiator will return [406 Not Acceptable](https://github.com/felipeleusin/hapi-negotiator/blob/master/lib/index.js#L29) so it makes this handler redundant, but Hapi requires there to be a handler so there needs to be something here :(